src/app/components/tabs/tabs-demo.component.html

```diff
@@ -11,3 +11,15 @@
     This tab cannot be closed.
   </sky-tab>
 </sky-tabset>
+
+<h3>Tabs example with counter</h3>
```

Comment:
I had the same thoughts. I just matched the old Sky UX 1 page for consistency as a starting place. What do you say to running a PR by them and then they can suggest changes here. Otherwise, Id just be guessing.